src/App.vue

```diff
@@ -1,32 +1,24 @@
 <template>
-  <div id="app">
-    <div id="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </div>
-    <router-view />
-  </div>
+  <v-app>
+    <navbar-layout />
+    <v-main class="grey lighten-3">
+      <v-container fluid>
+        <router-view />
+      </v-container>
+    </v-main>
+  </v-app>
 </template>
 
-<style lang="scss">
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-}
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
 
-#nav {
-  padding: 30px;
+import NavbarLayout from "@/components/NavBar/NavbarLayout.vue";
```

Comment:
@JuanJosePG When we name a directory of a component, the directory must be named with camelCase, and the component with PascalCase. Please, to rename the folder, you can ask me or look in the discord message that I have write, because is possible have errors when the folders are renamed.

Reply:
We can talk a day and discuss it.